manager/config/nocrd/manager_secret_provider_patch.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        env:
+        - name: SECRET_PROVIDER_URL
+          value: "http://secret-provider.secret-provider.svc.cluster.local:5555/get-secret"
+        - name: SECRET_PROVIDER_ROLE
+          value: "demo"
```

Comment:
I did not go into that resolution as I based this on the `manager/config/default` if this is redundant then it should be handled regardless of this PR, if so then please open an issue and it will be handled separately.